webpack.js

```diff
@@ -59,14 +59,24 @@ module.exports = function (config, basePath, options) {
   }
 
   // Prepend the Candela loaders.
+  var gloPath = basePath + '/node_modules/glo/glo.js';
   config.module.loaders = [
     {
       test: /\.js$/,
       loader: 'babel-loader',
       query: {
         presets: ['es2015']
       },
-      include: includePaths
+      include: includePaths.concat(gloPath)
+    },
+    {
+      test: /\.js$/,
+      tag: 'glo',
+      loaders: [
+        'exports-loader?GLO',
+        'imports-loader?_=underscore&cola=webcola'
+      ],
+      include: gloPath
```

Comment:
We were thinking of making GLO do this itself so that we didn't need this loader. I'm ok with a separate issue if you don't want to tackle this now.